### ISSUE TITLE
Implement Single Telescope training capability

### ISSUE THREAD

**Issue by bryankim96** [bryankim96/ctalearn#7]
Currently the approach used for training single tel models is to attach a logits layer directly to an array-model CNN-block. It may be worthwhile to implement a way to train arbitrary single tel models without this restriction.

**Comment by aribrill** [comment]
As of v0.1.2, models are defined by the user, so it is now possible to train arbitrary single telescope models.
